javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -526,11 +526,13 @@ public static boolean hasInheritedMeta(AnnotationMirror anno) {
     public static EnumSet<ElementKind> getElementKindsForElementType(ElementType elementType) {
         switch (elementType) {
             case TYPE:
-                return EnumSet.of(
-                        ElementKind.CLASS,
-                        ElementKind.INTERFACE,
-                        ElementKind.ANNOTATION_TYPE,
-                        ElementKind.ENUM);
+                EnumSet<ElementKind> typeKinds = EnumSet.noneOf(ElementKind.class);
```

Comment:
I'm wondering whether it's worth to also extract this into a field, like is done for trees.
This method isn't on a hot path as far as I can tell, but having the symmetry and separate code might still improve readability of the code.
What do you think?

Reply:
This sounds reasonable.  I'm wondering how far I should take the symmetry.

`TreeUtils` has:
```
private static final Set<Tree.Kind> classTreeKinds
public static Set<Tree.Kind> classTreeKinds()
public static boolean isClassTree(Tree)
```

I could add these to `ElementUtils`:
```
private static final Set<ElementKind> classElementKinds
public static Set<ElementKind> classElementKinds()
public static boolean isClassElement(Element)
```

and then I could refactor the places currently doing `element.getKind().isClass() || element.getKind().isInterface()` to instead do `ElementUtils.isClassElement(element)`.

Does that sound worthwhile, or is that going too far?

Reply:
Oops, one minor correction.  I would need to specify `EnumSet` as the type of `ElementUtils.classElementKinds` if I wanted to return it from `AnnotationUtils.getElementKindsForElementType`.  That, or return `EnumSet.copyOf(ElementUtils.classElementKinds())` (which would take an [optimized path in `EnumSet.copyOf(...)`](https://github.com/openjdk/jdk/blob/911846b49b85d71f41ec3b4d019e8873e64aee50/src/java.base/share/classes/java/util/EnumSet.java#L170-L172), since the collection it's copying also happens to be an `EnumSet`).

Reply:
That sounds worthwhile. Using EnumSet sounds good and would probably also be good for the Tree.Kind sets. Also, we should make defensive copies to prevent somebody from modifying the definitions of these sets.
Feel free to open an issue for further cleanups that aren't relevant for this PR.
Thanks!

Reply:
Mind taking another look?  I did not end up changing the static set types from `Set` to `EnumSet` or making defensive copies, because that was spiraling out into too many unrelated changes (changing other similar sets in `TreeUtils`, changing internal callers to avoid the defensive copies, but realizing there are no external callers and so no one needs the defensive copy, and so on).  IMHO it's ok as is, but let me know if you disagree.